services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/long_running_tasks.py

```diff
@@ -59,7 +60,8 @@ async def task_create_service_containers(
     )
     shared_store.container_names = assemble_container_names(shared_store.compose_spec)
 
-    logger.debug("Validated compose-spec:\n%s", f"{shared_store.compose_spec}")
+    # it is useful to see what compose spec was used to start the containers
+    logger.warning("Validated compose-spec:\n%s", f"{shared_store.compose_spec}")
```

Comment:
This seems like an INFO, and thus the default loglevel should be info and this message should maybe be info

Reply:
Add "Info:" to the string of the log thx

Reply:
what about this one?

Reply:
That is not what I want. I want to have a warning level and also see this message. If I set the log level to `DY_SIDECAR_LOG_LEVEL=INFO` it gets too spammy.